File: 151003/Tolkanitsa/discussion/src/main/java/com/example/discussion/model/entity/Comment.java
package com.example.discussion.model.entity;

import jakarta.persistence.Table;
import jakarta.validation.constraints.Size;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.cassandra.core.cql.PrimaryKeyType;
import org.springframework.data.cassandra.core.mapping.PrimaryKeyColumn;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "tbl_comment")
public class Comment {
    @PrimaryKeyColumn(name = "country", type = PrimaryKeyType.PARTITIONED)
    private String country;

    @PrimaryKeyColumn(name = "issueid", ordinal = 0, type = PrimaryKeyType.CLUSTERED)
    private Long issueId;

    @PrimaryKeyColumn(name = "id", ordinal = 1, type = PrimaryKeyType.CLUSTERED)
    private Long id;



    private String content;
}

File: 151003/Klimovich/Discussion/Models/Comment.cs
using Cassandra.Mapping.Attributes;

namespace Discussion.Models
{
    [Table("tbl_Comment")]
    public class Comment
    {
        [ClusteringKey]
        public int id { get; set; }

        [ClusteringKey]
        public int tweetId { get; set; }
        public string content { get; set; }
        [PartitionKey]
        public string country { get; set; }
    }
}
